### DISCUSSION TITLE
Customisation and templates for LinkFree profile

### DISCUSSION THREAD

**Discussion by Ananya2001-an** [EddieHubCommunity/LinkFree#4246] [Ideas]
Inspired by the cool customisation features that are provided by Linktree, I feel we can do something similar for LinkFree as well. Maybe have some available theme templates that can be chosen by users when they create their json file to be added. Just throwing ideas. Any suggestions are appreciated. Nice UI works after all ;)

**Comment by Ananya2001-an** [comment]
@eddiejaoude would like to hear your thoughts on the same.

**Reply by eddiejaoude** [reply]
I think themes are a great idea, for example
- profile page links as cards similar to the search results style
- profile page not to use tabs but have an "app" style feel to it, with everything on the same page
- and also the standard color themes

**Reply by Ananya2001-an** [reply]
Maybe we can create some designs and get an idea as to what can work

**Reply by eddiejaoude** [reply]
Some sketches could be interesting 👍 

I don't want to rush to fast into this, as I think we need to do some component refactoring first to avoid conflicts

**Reply by Ananya2001-an** [reply]
We can get more people to collaborate on this and give their suggestions or designs same as what was done for the linkfree logo. That would be a great contribution. 💯

**Reply by Ananya2001-an** [reply]
> Some sketches could be interesting 👍 
> 
> I don't want to rush to fast into this, as I think we need to do some component refactoring first to avoid conflicts 

Yes I agree just some rough sketches for now can be good to atleast have a picture in mind for the future.

**Reply by Yushi5058** [reply]
I'm interested for this templates idea, sounds like a good idea ^^
